### ISSUE TITLE
Visualization of the 5th conv layer is a noise?

### ISSUE THREAD

**Issue by zapaishchykova** [facebookresearch/deepcluster#44]
Hallo!
First, thank you for your amazing work!

Secondly, I am wondering why am I receiving a noise from visualization a 5th conv layer? I am using pretrained AlexNet and imageNet-tiny dataset(10 k pics), first I run `main.sh` and then `gradient_ascent.sh`. I tried with default settings as well as tried to change number of training epochs, batch size and nothing really changed - I am still recieving noise in con layer 5. Do you have any suggestions about why it doesn't work?

Also a question about the parameter `--nmb_cluster` - why is it by default 10k and what is it's meaning?

**Comment by kynantan** [comment]
I've also experienced images that appear to be only noise when running ```gradient_ascent.sh``` on any model, including the pre-trained AlexNet and VGG16 models with unmodified code. (and thanks very much for providing this work :) )

**Comment by mathildecaron31** [comment]
Hi,
Thanks for your interest, the `--nmb_cluster` corresponds to the number of clusters. You can refer to our DeepCluster paper for more details.

**Comment by mathildecaron31** [comment]
You should play with the hyper-parameters to find a regime where the visualisations are good.
For example with the pre-trained deepcluster AlexNet, for conv1 using a learning rate of 3 and 30.000 iterations works well. For conv5, using a learning rate of 30 and 3.000 iterations gives nice images with the other parameters set to their default values.
